### PR TITLE
new port: BRRTools

### PR DESCRIPTION
**Pull request by cbodek**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.9.5
Xcode 6.2

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [X] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by kencu** [comment]
generally speaking, ports that don't use a configure phase have a lot of problems setting the correct macports-specified compiler, using all our compiler options, handling arch flags and universal builds, etc, etc, without a lot of additional environment variables being manually set in the Portfile.

There are hundreds of examples in the macports Portfile tree.

This port has all those problems:
{{{
make: Entering directory `/opt/local/var/macports/build/_Users_travis_build_macports_macports-ports_audio_BRRTools/BRRTools/work/Optiroc-BRRtools-9093180'
gcc --std=c99 -Wall -O3 -c src/brr.c -o .build/brr.o
gcc --std=c99 -Wall -O3 src/brr_decoder.c .build/brr.o -lm -o bin/brr_decoder
gcc --std=c99 -Wall -O3 src/spc_decoder.c .build/brr.o -lm -o bin/spc_decoder
gcc --std=c99 -Wall -O3 src/brr_encoder.c .build/brr.o -lm -o bin/brr_encoder
}}}

File: audio/BRRTools/Portfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+long_description    BRRTools are currently the most evolved tools to \
+                    convert between standard RIFF .wav format and SNES's \
+                    built-in BRR sound format.
+platforms           darwin
+
+checksums           rmd160  149ba55f45b996fb0b3548faeafa4b29ec02471c \
+                    sha256  7d09bcfc5b58491ea68700130349cc9583296847325680ece41d295d0860431f \
+                    size    14731
+                                        
+use_configure       no
```
Comment (mf2k) [inline audio/BRRTools/Portfile:18]: As @kencu was saying, you need to make sure you are [using the right compiler]. (https://trac.macports.org/wiki/UsingTheRightCompiler)

**Comment by kencu** [comment]
I often find this incantation works:
```
use_configure       no

variant universal   {}
build.env           CXX=${configure.cxx} \
                    "CXXFLAGS=${configure.cxxflags} [get_canonical_archflags cxx]" \
                    CC=${configure.cc} \
                    "CFLAGS=${configure.cflags} [get_canonical_archflags cc]" \
                    "LDFLAGS=${configure.ldflags} [get_canonical_archflags ld]"

```

**Comment by kencu** [comment]
I forget right now about the quotes -- it changes in MP 2.6 -- someone is sure to correct me if they should be the other way.

**Comment by kencu** [comment]
Oh - one other thing - we talked about setting these ENVARS automatically in base, and I opened a ticket about that once upon a time -- and I think Marcus has been working on a PortGroup to do that for now (and perhaps integrate that into base someday if it works). So that might make it easier at some point. Dunno if that PG is currently in useable shape.

**Comment by kencu** [comment]
looks like you're stuck. Want some help with this?

**Comment by kencu** [comment]
0dfde99ffa136ff93f169455c2bca5108e1255a9

**Pull request closed without merging**